Asterisk.2013/Asterisk.NET/Manager/ManagerReader.cs

```diff
@@ -133,9 +133,9 @@ private void mrReaderCallbback(IAsyncResult ar)
 				// \n - because not all dev in Digium use \r\n
 				// .Trim() kill \r
 				lock (((ICollection) lineQueue).SyncRoot)
-					while (!string.IsNullOrEmpty(mrReader.lineBuffer) && (idx = mrReader.lineBuffer.IndexOf('\n')) >= 0)
-					{
-						line = idx > 0 ? mrReader.lineBuffer.Substring(0, idx).Trim() : string.Empty;
+                    while (!string.IsNullOrEmpty(mrReader.lineBuffer) && (idx = mrReader.lineBuffer.IndexOf('\n')) >= 0)

```

Comment:
Any reason why the indentation has changed here?

Reply:
Nop, i revert it back.
It was a mistake as i didn't notice the fix #140 already merged

Reply:
Visual Studio is picky and only want to use spaces, so any line you as much as touch gets `\t` replaced with `    ` (spaces). And for some reason GitHub uses some way long tab indention in the code views.

I would say to just ignore it. Maybe we should make a major "`\t` to `    `" replacement at some point to clean this up everywhere. No need to worry about it in this pull request though.
@skrusty just check the "Hide whitespace changes" option, it helps a little.
![image](https://user-images.githubusercontent.com/2676134/43006559-4423a758-8c36-11e8-8670-03f3223fd615.png)